modules/porous_flow/doc/content/modules/porous_flow/multiapp_fracture_flow_diffusion.md

```diff
@@ -0,0 +1,142 @@
+
+The two variables, $T_{f}$ and $T_{m}$, are called `frac_T` and `matrix_T` in the MOOSE input files.  Each obeys a diffusion equation, with heat transfer between the two variables, as written in the [eqn.coupled.basic].  The heat-transfer coefficient, $h$, may be written
+
+\begin{equation}
+\label{eqn.suggested.h}
+h = \frac{h_{\mathrm{s}}\lambda_{\mathrm{m}}^{nn} (L_{\mathrm{right}} + L_{\mathrm{left}})}{h_{\mathrm{s}}L_{\mathrm{right}}L_{\mathrm{left}} + \lambda_{\mathrm{m}}^{nn}(L_{\mathrm{right}} + L_{\mathrm{left}})} \ ,
+\end{equation}
+
+When using a MultiApp, the fracture appears as a set of Dirac sources in the matrix subdomain:
+
+\begin{equation}
+0 = c_{m}\dot{T}_{m} - \nabla(\lambda_{m}\nabla T_{m}) - H\delta(f) \ ,
+\end{equation}
+
+where
+
+\begin{equation}
+H = h(T_{f} - T_{m}) \ .
+\end{equation}
+
+In the MultiApp approach, $H$ is generated as an `AuxVariable` by the `fracture` App, so exists only in the `fracture` subdomain.  It is then passed to the `matrix` App, and applied as a `DiracKernel`.
+
+The boundary conditions are "no flow", except for the very left-hand side of the fracture domain, where temperature is fixed at $T_{f} = 1$.  The initial conditions are $T_{m} = 0 = T_{f}$.
+
+Each of the heat capacities are unity, $c_{m} = 1 = c_{f}$, the conductivity in the fracture is $\lambda_{f} = 1.0$, and is $\lambda_{m} = 10^{-3}$ in the matrix.  The fracure has aperture $a=10^{-2}$.
+
+Assume that $h_{s} = 0.02$.  This is the heat-transfer coefficient to use in the conforming case, since the matrix nodes align exactly with the fracture.  [eqn.suggested.h] may be used for the nonconforming case.  In this situation $L_{\mathrm{left}} = 0.1 = L_{\mathrm{right}}$ and $\lambda_{m}^{nn} = 10^{-3}$, so $h = 0.01$.
+
+Each simulation runs with `end_time = 100`.
+
+
+## No MultiApp: the benchmark
+
+In the conforming case, a MultiApp approach need not be taken, and the Kernels (given by [eqn.coupled.basic]) are:
+
+!listing fracture_diffusion/no_multiapp.i block=Kernels
+
+Evaluating the `fromFracture` heat transfer Kernel only on `block = fracture` implements the Dirac delta function $\delta(f)$ in [eqn.coupled.basic].  The matrix temperature is shown in [no_multiapp_matrix_T]
+
+!media porous_flow/examples/multiapp_flow/no_multiapp_matrix_T.png
+	style=width:60%;margin:auto;padding-top:2.5%;
+	id=no_multiapp_matrix_T
+	caption=Benchmark matrix temperature at the end of simulation
+
+The solution of this problem is [exactly solvable](https://github.com/idaholab/moose/blob/master/modules/misc/doc/fracture_flow) in terms of error functions, and by choosing appropriate time-step and element sizes, MOOSE produces the expected result.  Some examples of how MOOSE's solution depends upon time-step size are shown in [frac_no_multiapp_frac_T].  Evidently, reducing the time-step below 10.0 does not impact the solution very much.  Hence, the solution using $\Delta t = 0.125$ is used as the *benchmark* for the remainder of this section.
```

Comment:
is Delta t =0.125 correct.  Its not one of the dt's used in the figure

Reply:
I reference everything against this dt=0.125 "benchmark", so dt=0.125 doesn't appear in the figs. I've now changed the doco in a few places to emphasize this, eg `L2 error of each approach (with respect to the $\Delta t = 0.125$ benchmark) to...`